README.md

```diff
@@ -1,2 +1,46 @@
 # wg-meta
-An apporach to add metadata to the main wireguard config
+An apporach to add metadata to the main wireguard config, written in Perl
+
+## Configuration
+There is only one config file: `wg-meta.contract` which defines the additional (meta-) attributes:
+```text
+# ATTRIBUTE,    IS_OPTIONAL
```

Comment:
I would suggest to use `yaml` since it is human friendly and will let you extend things easily l8r ... 

```yaml
attribute:
   name:
       mandatory: true
   ... 
```      

ideally provide a json schema description of the config format

Reply:
Thank you for you feedback. I totally agree with your suggestion. However, I decided against defining each attribute separately since this would mean that the user has to adjust the schema when adding a new attribute (at least as far as I understand the way how these JSON schemas are defined)